src/thunderbots/software/ai/hl/stp/evaluation/indirect_chip.h

```diff
@@ -0,0 +1,144 @@
+#include "ai/world/world.h"
+#include "geom/angle.h"
+#include "geom/point.h"
+#include "util/parameter/dynamic_parameters.h"
+
+/**
+ * The indirect chip and chase target evaluation function returns a target point for the
+ * chipper and chaser.
+ * Other related evaluation functions are also included in this file.
+ */
+namespace Evaluation
+{
+    typedef Point Vector2;
+    template <size_t N>
+    using Poly     = std::array<Vector2, N>;
+    using Triangle = Poly<3>;
+
+    /**
+     * Returns the target point that the chipper and chaser will chip and chase at.
+     *
+     * Given the enemy robots' positions, filters and creates a vector of
+     * triangles to be used as a parameter. The target point is where ball will land
+     * according to chipping calibration, as well as where the chaser will meet the ball
+     * at.
+     *
+     * @param world The world in which we want to find the target point
+     *
+     * @return Point to chip and chase at; If null, the target triangles were empty
+     */
+    std::optional<Point> findTargetPointForIndirectChipAndChase(const World& world);
+
+    /**
+     * Returns the target point that the chipper and chaser will chip and chase at.
+     *
+     * Given the vector of triangles without enemy robots, determines if the triangles are
+     * empty and if the largest triangle is within reach. If not within reach, scale the
```

Comment:
do we mean "if there are any triangles"? Or is this referring to there being robots in the triangles?

Reply:
The vector of triangles (input to this function) should be "filtered" by now from the wrapper function so that there are no enemy robots in any triangles.  I hope that answers your question! Which line in specific are you referring to?

Reply:
I'm not too sure honestly........... resolving